src/hash-table.js

```diff
@@ -7,6 +7,27 @@ class HashTable {
     this.storage = new LimitedArray(this.limit);
     // Do not modify anything inside of the constructor
   }
+  insert(data, ...theArgs) {
```

Comment:
The spread operator use here isn't achieving anything in your implementation, since you only take `theArgs[0]` and discard the rest that may have been passed in.